### ISSUE TITLE
Missing donation data for BIDEN FOR PRESIDENT (C00703975) Sep 1 - 15

### ISSUE THREAD

**Issue by EliasWehbe** [fecgov/FEC#10451]
There are only 272 entries here. Why is that?

https://www.fec.gov/data/receipts/individual-contributions/?committee_id=C00703975&min_date=09%2F01%2F2020&max_date=09%2F15%2F2020

**Comment by PaulClark2** [comment]
Thank you for contacting the Federal Election Commission. Do you have specific contributions that you think are missing? 272 matches our source data. Here are individual contributions counts by day for the month of September. 

Date	|	Contribution count	|		|		|
----	|	----	|	----	|	----	|
9/1/2020	|	8	|		|		|
9/2/2020	|	2	|		|		|
9/3/2020	|	2	|		|		|
9/8/2020	|	101	|		|		|
9/9/2020	|	48	|		|		|
9/10/2020	|	33	|		|		|
9/11/2020	|	19	|		|		|
9/12/2020	|	2	|		|		|
9/13/2020	|	5	|		|		|
9/14/2020	|	18	|		|		|
9/15/2020	|	34	|	272	|	9/1-9/15 subtotal	|
9/16/2020	|	20	|		|		|
9/17/2020	|	19	|		|		|
9/18/2020	|	15	|		|		|
9/19/2020	|	2	|		|		|
9/20/2020	|	28,656	|		|		|
9/21/2020	|	14,234	|		|		|
9/22/2020	|	9,161	|		|		|
9/23/2020	|	9,896	|		|		|
9/24/2020	|	8,962	|		|		|
9/25/2020	|	5,169	|		|		|
9/26/2020	|	1,986	|		|		|
9/27/2020	|	27,627	|		|		|
9/28/2020	|	18,622	|		|		|
9/29/2020	|	20,090	|		|		|
9/30/2020	|	21,921	|	166,652	|	September total	|

**Comment by EliasWehbe** [comment]
Yes that doesn't make sense. Only 272 donations in the first half of September is off. There must be missing data there, especially if you compare to the second half of September and first half of August.

**Comment by PaulClark2** [comment]
There was an error when loading the transactions into our database. The transaction were missing a foreign key. We've corrected this issue. You should see on the website all September transactions tomorrow morning.

**Comment by EliasWehbe** [comment]
Great thanks.

**Comment by EliasWehbe** [comment]
Also do you know when the next batch will be release? Thanks.

**Comment by PaulClark2** [comment]
M10 individual contribution counts by day

Date	|	Transaction count
---	|	---
9/1/2020	|	21,426
9/2/2020	|	20,379
9/3/2020	|	30,827
9/4/2020	|	33,443
9/5/2020	|	19,880
9/6/2020	|	34,397
9/7/2020	|	19,720
9/8/2020	|	37,209
9/9/2020	|	30,301
9/10/2020	|	57,237
9/11/2020	|	16,777
9/12/2020	|	18,249
9/13/2020	|	48,042
9/14/2020	|	52,354
9/15/2020	|	44,942
9/16/2020	|	46,618
9/17/2020	|	38,490
9/18/2020	|	32,750
9/19/2020	|	46,903
9/20/2020	|	30,824
9/21/2020	|	41,793
9/22/2020	|	21,477
9/23/2020	|	28,693
9/24/2020	|	38,163
9/25/2020	|	26,770
9/26/2020	|	38,716
9/27/2020	|	29,625
9/28/2020	|	35,132
9/29/2020	|	102,149
9/30/2020	|	117,449

**Comment by PaulClark2** [comment]
The 12G (pre-general report) has been processed. The 12G is only includes financial activity from October 1 through 14. The next regularly scheduled report is the 30G (post-general) which is due December 3. 

Date	|	Transaction count
---	|	---
10/1/2020	|	50,246
10/2/2020	|	28,436
10/3/2020	|	23,880
10/4/2020	|	57,583
10/5/2020	|	49,934
10/6/2020	|	44,683
10/7/2020	|	121,763
10/8/2020	|	59,815
10/9/2020	|	44,086
10/10/2020	|	44,503
10/11/2020	|	44,533
10/12/2020	|	60,370
10/13/2020	|	104,743
10/14/2020	|	96,534
